esx_service/vmdk_ops.py

```diff
@@ -606,6 +606,13 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     elif cmd == "remove":
         response = removeVMDK(vmdk_path)
+        # remove succeed, remove infomation of this volume from volumes table
```

Comment:
The reason we cannot insert it into "volumes" table is the volume is created by a VM which is not associate with any tenant.  volumes table need key like (tenant_uuid, datastore, volume_name). 

I don't understand what do you mean "how does it impact the location of DB code ?". The function to add/remove volume from "volumes" table is in auth.py.